### PR TITLE
feat(content-banner): Fix container for banner inside tab panel - OEL-1935

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/themes/default/src/scss/_featured-media.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.bcl-featured-image {
```
Comment (planctus) [inline src/themes/default/src/scss/_featured-media.scss:1]: How is this related..?

Comment (tibi2303) [reply src/themes/default/src/scss/_featured-media.scss]: weird.. i don't know why is this here... deleted

**Review by planctus — APPROVED** [review]
